src/zzz_od/gui/view/devtools/intel_manage_interface.py

```
@@ -0,0 +1,1273 @@
+
+        # 属性类型下拉框
+        self.dmg_type_combo = ComboBox()
+        dmg_type_items = [ConfigItem(enum.value, enum.name) for enum in DmgTypeEnum]
+        self.dmg_type_combo.set_items(dmg_type_items)
+
+        # 稀有度下拉框
+        self.rare_type_combo = ComboBox()
+        rare_type_items = [ConfigItem(enum.value, enum.name) for enum in RareTypeEnum]
+        self.rare_type_combo.set_items(rare_type_items)
+
+        # 设置当前值（优先使用英文标识，回退到中文显示值）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)